docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,63 @@
+- Compass UI, where you create connections between Applications, Runtimes, and Scenarios
+- Kyma Console UI, where you manage resources used in your Application, such as services, lambdas, and bindings
+
+## Prerequisites
+
+Use the [HTTP DB Service](https://github.com/kyma-project/examples/tree/master/http-db-service) Application to go through this tutorial. Prepare the following:
+- [File](./assets/http-db-service-deployment.yaml) that contains the HTTP DB Service definition and deployment
+- HTTP DB Service [API spec](./assets/http-db-api-spec.yaml)
+- [`call-order-service`](./assets/lambda.yaml) lambda function
+- Kyma cluster with the Compass module enabled
+
+>**NOTE:** Read [this](#installation-enable-compass-in-kyma-compass-as-a-central-management-plane) document to learn how to install Kyma with the Compass module enabled.
+
+## Steps
+
+### Connect the external Application
+
+1. Log in to the Kyma Console and create the `test` Namespace by clicking the **Add new namespace** button.
+2. In the **Overview** tab in your Namespace, click the **Deploy new resource** button and select the file with HTTP DB Service and deployment to connect the Application.
+
+### Register your Application in the Compass UI
+
+1. Go **Back to Namespaces** at the left top of the page and select the **Compass** tab in the left navigation panel, which navigates you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant.
+2. In the **Runtimes** tab, there is already `kymaruntime` that you can work on to complete this tutorial. This is the default Runtime that is already assigned to the `DEFAULT` scenario. You can also create a new Runtime by clicking the **Create runtime** button. After creating the Runtime, navigate to it, click **Edit**, and assign it to the scenario you want to work on. To create scenarios, go to the **Scenarios** tab and click the **Create Scenario** button. While creating a new scenario, you can assign Runtimes and Applications to it from the drop-down lists.
+3. Navigate to the **Application** view and click **Create Application** to register your Application in Compass. For the purpose of this tutorial, name your Application `test-app`. By default, your Application is assigned to the `DEFAULT` scenario.
+4. Click on `test-app` in the **Applications** view and add the API spec of the HTTP DB Service. To do so, click the **+** button in **API Definitions** section. Fill in all the required fields,  click the **Add specification** button below and upload the HTTP DB Service API spec file. Paste the URL to your Application in the **Target URL** field. In case of HTTP DB Service, proceed with the next steps before filling this field. In the **Credentials** tab, choose `None` as credential type from the drop-down list since, for the purpose of this tutorial, there is no need to secure the connection.
```

Comment:
how I can do that: Paste the URL to your Application in the **Target URL** field. ?